Gemfile

```diff
@@ -2,16 +2,13 @@
 # If you modify this file manually all comments and formatting will be lost.
 
 source "https://rubygems.org"
-gem "logstash-core", "5.0.0.dev", :path => "./logstash-core"
-gem "logstash-core-event-java", "5.0.0.dev", :path => "./logstash-core-event-java"
-gem "logstash-core-plugin-api", "1.0.0", :path => "./logstash-core-plugin-api"
+gem "logstash-core", "5.0.0.dev", :path => "vendor/local_gems/ee2f9cf4/logstash-core-5.0.0.dev-java"
```

Comment:
Same thing here, this looks like some kind of leftover from running all plugins test?